### PR TITLE
Animation Panel: Fix pan regression and add translation safety

### LINKED ISSUE #7123: Animations: Missing internationalization
## Bug Description

Some strings in `defaultAnimationProps.js` aren't translatable, which seems like a bug.

https://github.com/google/web-stories-wp/blob/07c07c1af196da69ef13dfeceb2af87d83fc2ee6/assets/src/animation/parts/defaultAnimationProps.js#L54-L89
## Expected Behaviour

All user-facing strings should be localizable (translatable)

## Additional Context

- Plugin Version: 1.6.0-alpha
- WordPress Version: 5.7
- Operating System: macOS
- Browser: Chrome

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
Part of fixit week. Pascal noticed some untranslated stuff in the default animation props, so the ticket was to look into that.

## Summary
While looking into the animation props it seems none of the untranslated strings were user facing so nothing really needed to be updated. Added some translations to labels in there just for extra safety, but none of them were user facing to begin with so 🤷 

While I was in here tho, I noticed all the pan animation directions were disabled for bg elements. It seems elements with type `media` have been updated to be of type `video` , `image` , or `gif` which caused a regression in the offset testing. I just updated it to accept the new more refined media types and now it works as it did before.

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
Pan direction for bg element animations should now work again.

## Testing Instructions
Go to story editor. Make an image a bg element on one page, a gif a bg element on another & a video bg element on another. See that the pan directions are properly enabling and disabling for the bg animation options on each type of bg element.

### QA
Follow testing instructions described above.

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7123

### CODE REVIEW

**Review by BrittanyIRL — APPROVED** [review]
Good catch on the bg pans!

**Review by samwhale — APPROVED** [review]
noice

**Review by embarks — APPROVED** [review]

**Comment by csossi** [comment]
Pan and Zoom behaviour changes to be the same as Zoom In

- added image background
- checked Pan And Zoom - worked fine
- enlarged the image (needed to be enlarged to allow Pan Up, Pan Down, and Zoom In to work)
- Pan Up, Pan Down, and Zoom In work fine - but now Pan And Zoom (which was fine before enlarging) changes to be the same as Zoom In

File: assets/src/edit-story/components/panels/design/animation/animation.js [hunk 1]
```diff
@@ -181,7 +181,7 @@ function AnimationPanel({
   const disabledTypeOptionsMap = useMemo(() => {
     if (selectedElements[0]?.isBackground) {
       const hasOffset =
-        'media' === selectedElements[0].type &&
+        ['media', 'image', 'video', 'gif'].includes(selectedElements[0].type) &&
```
Comment (swissspidy) [inline assets/src/edit-story/components/panels/design/animation/animation.js]: I don't think `media` is an actual element type 🤔

**Pull request merged**